SU2_CFD/include/solvers/CScalarSolver.inl

```diff
@@ -410,8 +410,10 @@ void CScalarSolver<VariableType>::CompleteImplicitIteration(CGeometry* geometry,
     CompletePeriodicComms(geometry, config, iPeriodic, PERIODIC_IMPLICIT);
   }
 
-  InitiateComms(geometry, config, SOLUTION_EDDY);
-  CompleteComms(geometry, config, SOLUTION_EDDY);
+  if (config->GetKind_Turb_Model() != TURB_MODEL::NONE) {
+    InitiateComms(geometry, config, SOLUTION_EDDY);
+    CompleteComms(geometry, config, SOLUTION_EDDY);
+  }
```

Comment:
I think you always need the comms, but the eddy-viscosity part should not be needed, even for turbulence solvers
```suggestion
  InitiateComms(geometry, config, SOLUTION);
  CompleteComms(geometry, config, SOLUTION);
```

Reply:
👍 Yep I think you are right: The muT computation in Turb(SA,SST)::Post does not contain any `GetDomain(iPoint)` break therefore the correct muT should be computed even for the halo cells. And that muT comm is the only difference between SOLUTION_EDDY and SOLUTION. So I will change that as that SOLUTION comms should be done for the species solver

I will do so without this suggestion as it will introduce residual changes in the reg tests and I prefer to have that contained in one commit